tutorials/lb-firewall-haproxy-pfsense/index.mdx

```diff
@@ -304,7 +304,7 @@ Before configuring High Availability, rules to allow `pfsync` traffic have to be
 
 ## Configuring failover IPs
 
-Two [failover IPs](/dedibox-console/my-account/how-to/outsource-ip-failover) are being used as an endpoint for HAProxy (one by each instance). In order to automatize the configuration in case of unavailability of one of the servers, the following workflow is needed:
+Two [failover IPs](/dedibox-console/my-account/how-to/outsource-failover-ip) are being used as an endpoint for HAProxy (one by each instance). In order to automatize the configuration in case of unavailability of one of the servers, the following workflow is needed:
```

Comment:
```suggestion
Two [failover IPs](/dedibox-console/my-account/how-to/outsource-failover-ip) are being used as an endpoint for HAProxy (one by each Instance). In order to automatize the configuration in case of unavailability of one of the servers, the following workflow is needed:
```